src/Delivery/DynamicEntry.php

```diff
@@ -69,9 +69,15 @@ public function getContentType()
         return $this->sys->getContentType();
     }
 
+    /**
+     * @param  string $name
+     * @param  array  $arguments
+     *
+     * @return mixed
+     */
     public function __call($name, $arguments)
     {
-        if (substr($name, 0, 3) !== 'get') {
+        if (0 !== strpos($name, 'get')) {
```

Comment:
So _name_ can be `getter`, `target` or `together`? I think I liked the former solution better ...